web/source/kmwbase.ts

```diff
@@ -159,12 +159,12 @@ namespace com.keyman {
       this.osk = this['osk'] = null;
 
       // Load properties from their static variants.
-      this['build'] = com.keyman.environment.BUILD;
+      this['build'] = Number.parseInt(com.keyman.KEYMAN_VERSION.VERSION_PATCH, 10);
       this.srcPath = KeymanBase._srcPath;
       this.rootPath = KeymanBase._rootPath;
       this.protocol = KeymanBase._protocol;
 
-      this['version'] = com.keyman.environment.VERSION;
+      this['version'] = com.keyman.KEYMAN_VERSION.VERSION_RELEASE;
```

Comment:
Once we have KEYMAN_VERSION, we really should also publish a full version string in KMW, e.g. this['version_with_tag'] = ...?